source/interface.cc

```diff
@@ -268,6 +268,15 @@ Interface<dim,spacedim,n_components,LAC>::initialize_data(const typename LAC::Ve
                                                           const double t,
                                                           const double alpha) const
 {
+  std::cout << "CALL" << std::endl;
+
+  if(this->old_t < t)
+  {
+    this->old_t = t;
+    this->old_solution = &solution;
```

Comment:
~~~
this->old_solution.reinit(solution, true);
this->old_solution = solution;
~~~